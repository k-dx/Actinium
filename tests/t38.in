let a = [1, 2, 3];
let i = 0;
let b = (a = [10, 20, 30])[i = 2];
print i; // => 2
print b; // => 30
print a; // => [10, 20, 30]