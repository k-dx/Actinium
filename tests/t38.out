2
30
[10, 20, 30]